pynicotine/gtkgui/settingswindow.py

```diff
@@ -2463,6 +2463,8 @@ def get_player(self):
             player = "lastfm"
         elif self.NP_mpris.get_active():
             player = "mpris"
+        elif self.NP_listenbrainz.get_active():
+            player = "listenbrainz"
```

Comment:
While this is a good option for users, there may be many other options desired in the future. We should provide a /plugins/ option so that third-party developers don't need to modify the core.